rules/Multifactor-Google-Authenticator-Do-Not-Rename.js

```diff
@@ -1,5 +1,6 @@
 function (user, context, callback) {
     var AUTHENTICATOR_LABEL = 'MOJ Analytical Platform (dev)';
+    var CONNECTION = user.identities[0].connection;
```

Comment:
Is this assuming that the user has only 1 identity, and/or that the first identity is the one with the connection?

Reply:
Correct. All our Auth0 users have 1 identity and that's the user used to log in via that connection.